maven-artifact/src/test/java/org/apache/maven/artifact/versioning/ComparableVersionTest.java

```diff
@@ -337,4 +337,20 @@ public void testReuse()
 
         assertEquals( "reused instance should be equivalent to new instance", c1, c2 );
     }
+
+    /**
+     * Test <a href="https://issues.apache.org/jira/browse/MNG-7644">MNG-7644</a> edge cases
+     * 1.0.0.RC1 < 1.0.0-RC2
+     */
+    public void testMng7644()
+    {
+        for ( String x : new String[]{ "alpha", "a", "beta", "b", "milestone", "m", "RC" } ) {
```

Comment:
You should add at least two arbitrary one as well.

Reply:
i suppose you mean like abc and def. will do

Reply:
done